docs/reference/feedstock-settings.md

```diff
@@ -0,0 +1,33 @@
+# Feedstock settings
+
+When `conda-smithy` rerenders the contents of the feedstock, it takes into account different sources of configuration:
+
+- `conda-forge.yml`
+- `recipe/conda_build_config.yaml`
+- `recipe/meta.yaml` (only the `skip: true` lines)
+- `.ci_support/migrations/*` (added by the autotick bot)
+
+:::caution
+Outside `recipe/`, the only file you are supposed to edit is `conda-forge.yml`.
+Everything else is automatically managed!
+:::
+
+## conda-forge.yml {#conda-forge-yml}
+
```

Comment:
As an observation, I think we should just go over bit-by-bit of the fields and what they generally mean (in a block fashion), and add more depth to it under an advanced page.... what do you think?

Reply:
Ideally this is auto-generated from the conda-smithy schema, but such a thing doesn't exist yet :/